oak/server/dev/dev_oak_loader.cc

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2018 The Project Oak Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "dev_oak_loader.h"
```

Comment:
Should this be `"oak/server/dev/dev_oak_loader.h"` ?

Reply:
I thought we always used `#include "file.h"` for `file.cc` (without a full path):
https://github.com/project-oak/oak/blob/664f0d6ee80ab805b72a461b1b879f5fdca0de2a/oak/server/dev/dev_oak_manager.cc#L17
https://github.com/project-oak/oak/blob/664f0d6ee80ab805b72a461b1b879f5fdca0de2a/oak/server/asylo/enclave_server.cc#L17

Reply:
Seems like the [recommendation](https://docs.bazel.build/versions/master/bazel-and-cpp.html#include-paths) is to have paths rooted from where the `WORKSPACE` file is.

Reply:
Done